src/bp-templates/bp-nouveau/js/buddypress-activity-post-form.js

```diff
@@ -1937,6 +1937,7 @@ window.bp = window.bp || {};
 
 			getURL: function ( prefix, urlText ) {
 				var urlString   = '';
+				urlText         = urlText.replace(/&nbsp;/g, '');
```

Comment:
@bb-yudhisthir I think `trim` is the proper solution than `replace`. 

```Trim() removes all whitespace characters from the beginning and end of the string. That means spaces , tabs , new lines , returns , and other assorted whitespace characters. Replace() only replaces the designated characters with the given replacement.```

Reply:
trim() replaces the whitespace but doesn't replace the "&nbsp;" chars, I have already tested it http://somup.com/crQUfPr9RA
Also we are already using the trim in subsequent function - loadURLPreview() but not working for "&nbsp" chars